tests-behat/bootstrap/Context.php

```diff
@@ -386,6 +386,9 @@ public function iSelectValueInLookup($arg1, $arg2)
 
         // hide dropdown and wait till fully closed
         $this->getSession()->executeScript('$("#' . $lookupElem->getAttribute('id') . '").dropdown("hide");');
+        $this->jqueryWait();
+        // for unknown reasons, dropdown very often remains visible in CI, so hide twice
+        $this->getSession()->executeScript('$("#' . $lookupElem->getAttribute('id') . '").dropdown("hide");');
```

Comment:
This was the final CI issue - @ibelar please report to FUI if not related with https://github.com/atk4/ui/issues/1521 , ie. fixed by https://github.com/fomantic/Fomantic-UI/pull/1725